blockchain/pool.go

```diff
@@ -363,23 +363,23 @@ func (pool *BlockPool) sendError(err error, peerID p2p.ID) {
 	pool.errorsCh <- peerError{err, peerID}
 }
 
-// unused by tendermint; left for debugging purposes
-func (pool *BlockPool) debug() string {
-	pool.mtx.Lock()
-	defer pool.mtx.Unlock()
-
-	str := ""
-	nextHeight := pool.height + pool.requestersLen()
-	for h := pool.height; h < nextHeight; h++ {
-		if pool.requesters[h] == nil {
-			str += fmt.Sprintf("H(%v):X ", h)
-		} else {
-			str += fmt.Sprintf("H(%v):", h)
-			str += fmt.Sprintf("B?(%v) ", pool.requesters[h].block != nil)
-		}
-	}
-	return str
-}
+// for debugging purposes
+// func (pool *BlockPool) debug() string {
```

Comment:
I wish there was a way to tell the linter: ignore this function (like for instance with clippy in rust).

Reply:
Wait, doesn't `nolint:` work for golangci, too?

Reply:
It does. Do you think we should add such labels instead of commenting?

Reply:
Yes, (in the case of deadcode) if we consider the code to be useful, I would prefer a nolint there. If it's clear that it isn't useful and won't be used (for debugging or alike), I would prefer to delete it (even better).

Reply:
Done

Reply:
Awesome! This also makes it easier to search for these places.